kommunicateui/src/main/java/com/applozic/mobicomkit/uiwidgets/conversation/activity/FullScreenImageActivity.java

```diff
@@ -104,8 +123,16 @@ public void onResourceReady(@NonNull Bitmap resource, @Nullable Transition<? sup
 
             if (message != null && message.getFilePaths() != null && !message.getFilePaths().isEmpty()) {
                 try {
-                    Bitmap imageBitmap = ImageUtils.decodeSampledBitmapFromPath(message.getFilePaths().get(0));
-                    mediaImageView.setImageBitmap(imageBitmap);
+                    if(message.getFileMetas().getContentType().contains("gif")) {
+                        Glide.with(this)
+                                .asGif()
+                                .load(message.getFilePaths().get(0))
+                                .into(gifImageView);
+                    }
+                    else {
+                        Bitmap imageBitmap = ImageUtils.decodeSampledBitmapFromPath(message.getFilePaths().get(0));
```

Comment:
Are we checking this getFilePaths().length anywhere? in this case if message.getFilePaths() is empty , this line will produce crash

Reply:
We are already doing a check:
if (message != null && message.getFilePaths() != null && !message.getFilePaths().isEmpty())